app/controllers/courses_controller.rb

```diff
@@ -208,6 +208,17 @@ def set_timeline_enabled
     end
   end
 
+  def set_course_edits_enabled
```

Comment:
Doesn't look like this method gets called.

Reply:
Fixed, thank you!